bigquery/tables.js

```diff
@@ -252,7 +266,7 @@ cli
   .command('create <dataset> <table>', 'Create a new table in the specified dataset.', {}, function (options) {
     program.createTable(utils.pick(options, ['dataset', 'table']), utils.makeHandler());
   })
-  .command('list <dataset>', 'List tables in the specified dataset.', {}, function (options) {
+  .command('tables <dataset>', 'List tables in the specified dataset.', {}, function (options) {
```

Comment:
I was thinking of having `list` for listing tables, since the sample is `tables.js`, and `browse` for the rows, which matches the sample description "Browse a table.".

```
node tables list my_dataset
node tables browse my_dataset my_table
```